### ISSUE TITLE
Error response from daemon: Get "https://registry-1.docker.io/v2/":

### ISSUE THREAD

**Issue by bogordesaincom** [docker/for-mac#6621]
Error response from daemon: Get "https://registry-1.docker.io/v2/": dialing registry-1.docker.io:443 static system has no HTTPS proxy: connecting to 44.205.64.79:443: dial tcp 44.205.64.79:443: i/o timeout

### Docker Info 

```
Client:
 Context:    default
 Debug Mode: false
 Plugins:
  buildx: Docker Buildx (Docker Inc., v0.9.1)
  compose: Docker Compose (Docker Inc., v2.13.0)
  dev: Docker Dev Environments (Docker Inc., v0.0.5)
  extension: Manages Docker extensions (Docker Inc., v0.2.16)
  sbom: View the packaged-based Software Bill Of Materials (SBOM) for an image (Anchore Inc., 0.6.0)
  scan: Docker Scan (Docker Inc., v0.22.0)

Server:
 Containers: 0
  Running: 0
  Paused: 0
  Stopped: 0
 Images: 0
 Server Version: 20.10.21
 Storage Driver: overlay2
  Backing Filesystem: extfs
  Supports d_type: true
  Native Overlay Diff: true
  userxattr: false
 Logging Driver: json-file
 Cgroup Driver: cgroupfs
 Cgroup Version: 2
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
 Swarm: inactive
 Runtimes: io.containerd.runc.v2 io.containerd.runtime.v1.linux runc
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: 770bd0108c32f3fb5c73ae1264f7e503fe7b2661
 runc version: v1.1.4-0-g5fd4c4d
 init version: de40ad0
 Security Options:
  seccomp
   Profile: default
  cgroupns
 Kernel Version: 5.15.49-linuxkit
 Operating System: Docker Desktop
 OSType: linux
 Architecture: aarch64
 CPUs: 4
 Total Memory: 3.841GiB
 Name: docker-desktop
 ID: TIYL:LPT6:2B2E:DWKK:DPGZ:JSSP:MMID:JZZ6:PX6V:QWLO:FDQ4:CY6D
 Docker Root Dir: /var/lib/docker
 Debug Mode: false
 HTTP Proxy: http.docker.internal:3128
 HTTPS Proxy: http.docker.internal:3128
 No Proxy: hubproxy.docker.internal
 Registry: https://index.docker.io/v1/
 Labels:
 Experimental: false
 Insecure Registries:
  hubproxy.docker.internal:5000
  127.0.0.0/8
 Live Restore Enabled: false
```

**Comment by pburnwal-atlassian** [comment]
Hi, I am facing the same issue. Can someone please advice? Thanks.

**Comment by goodtune** [comment]
Glad to see I am not alone!

```
Post "https://login.docker.com/oauth/token": dialing login.docker.com:443 static system has no HTTPS proxy: connecting to 104.16.171.253:443: dial tcp 104.16.171.253:443: i/o timeout
```

**Comment by gmaquinay** [comment]
Same for me, but only when I'm connecting via hotspot.

**Comment by goodtune** [comment]
I was having some issues in other applications which were narrowed down to IPv6. Once I allowed ICMPv6 traffic through my firewall, I was able to use the other services and my Docker Desktop login worked fine also.

I would suggest using http://ipv6-test.com/ and if you have an IPv6 stack, make sure the ICMP tests pass. This was the only thing I changed to go from consistent failure to immediate success.

**Comment by vicary** [comment]
@goodtune Hi, I am having the same issue. My ISP does not support v6 stacks yet, is it possible for my local machine to stop pulling images via IPv6 altogether?

**Comment by davibusanello** [comment]
Same problem, any solution?

**Comment by aurishhammadhafeez** [comment]
I faced the same problem, changing the internet connection worked.

**Comment by vicary** [comment]
@aurishhammadhafeez would you please elaborate, what have you changed? We are eager to solve this issue in our pipelines.

**Comment by ZDiamonDZ** [comment]
Same issue:
Error response from daemon: Head "https://registry-1.docker.io/v2/elkozmon/zoonavigator/manifests/latest": dialing registry-1.docker.io:443 static system has no HTTPS proxy: connecting to 34.205.13.154:443: dial tcp 34.205.13.154:443: i/o timeout

**Comment by aurishhammadhafeez** [comment]
@vicary I was facing this on my development machine so I just changed my ISP and it worked.
Maybe Docker implemented some new network policies that some ISP providers might not have updated yet.

**Comment by lucasvscn** [comment]
**Docker Desktop version: 4.15.0 (93002)**

I'm having this issue while routing my 5G from iPhone into my Mac.

**Comment by DEEJCUK** [comment]
same issue for me, i have little snitch installed, i "turned off the network filter" in LS and it worked.

**Comment by ThomaciousD** [comment]
> same issue for me on Mac OSX 12.6.2 with latest docker desktop, i have little snitch installed, i "turned off the network filter" in LS and it worked.

This is the best answer that worked for me, after trying all the others.

**Comment by walliby** [comment]
I started getting this error yesterday and have tried everything. Has anyone found a solution other than changing ISPs?

Error response from daemon: Get "https://registry-1.docker.io/v2/": dialing registry-1.docker.io:443 static system has no HTTPS proxy: connecting to 3.216.34.172:443: dial tcp 3.216.34.172:443: i/o timeout

**Comment by yvdlima** [comment]
Updating Docker Desktop to v4.16.0 today solved the issue for me

**Comment by srivastavarobin** [comment]
For Mac OS v13.1, go to System Settings -> Network -> Firewall -> Options; and add Docker in the list to allow incoming connections. This solved it for me.

**Comment by bogordesaincom** [comment]
> 

yes is solved in version 

Docker Desktop 4.15.0 (93002) is currently the newest version available.

**Comment by fikriauliya** [comment]
I solve this issue by using 8.8.8.8 DNS. Probably my DNS server blocks this

**Comment by GeorgeZhukov** [comment]
> I solve this issue by using 8.8.8.8 DNS. Probably my DNS server blocks this

Thanks! Changing the dns to the google version helped me as well.

**Comment by profmcdan** [comment]
> http://ipv6-test.com/

This works for me also.

Thanks @fikriauliya

**Comment by arun-venkataswamy** [comment]
> For Mac OS v13.1, go to System Settings -> Network -> Firewall -> Options; and add Docker in the list to allow incoming connections. This solved it for me.

This solved it for me too.

**Comment by bikoevD** [comment]
For Mac OS v12.1, System -> Security & Privacy -> Full Disk Access and add Docker

**Comment by cnzexin** [comment]
> For Mac OS v12.1, System -> Security & Privacy -> Full Disk Access and add Docker

My Mac OS is Ventura 13.2.1 (22D68) , your method is ok for me. Thanks!

**Comment by maxhaice** [comment]
If someone didn't find the solution.
I'm not sure that something helped me.
I tried all ways, docker version 4.17.0.
I change docker theme to Dark and it earned!

Oh, sorry, **maybe** it was joke. I just did 'clean / purge data', 'reset to factory defaults' (i didn't change settings before, only experimental features) 
<img width="1200" alt="image" src="https://user-images.githubusercontent.com/50238959/225081127-d347d673-bde0-4024-91f3-b139a674e18b.png">

P.S. I didn't touch my mac for about a half of year, and docker was working before. (macOS Ventura)
